source/_docs/wordpress-phpstorm.md

```diff
@@ -1,64 +1,97 @@
 ---
-title: Configuring JetBrains PhpStorm IDE with WordPress on Pantheon
-description: Best practices and recommendations for building a WordPress site using JetBrains PhpStorm.
+title: Configuring PhpStorm to create WordPress Plugins on Pantheon
+description: Configure a local environment for building a WordPress plugin or theme using JetBrains PhpStorm.
 tags: [local]
 categories: [wordpress]
 ---
 ## Overview
 
 [JetBrains PhpStorm](https://www.jetbrains.com/phpstorm/) is a commercial PHP IDE that can be configured to work with WordPress sites, allowing users to easily build and maintain custom plugins.
 
-For detailed information, see [WordPress Development using PhpStorm](https://confluence.jetbrains.com/display/PhpStorm/WordPress+Development+using+PhpStorm). This document focuses on integrating and showcasing the PhpStorm workflow.
+This doc will set you up with a local environment in PhpStorm to create your custom plugins and themes. If you're looking to use PhpStorm to manage code for an entire site, [click here](/docs/wordpress-phpstorm/).
+
+For detailed information, see [WordPress Development using PhpStorm](https://confluence.jetbrains.com/display/PhpStorm/WordPress+Development+using+PhpStorm). This document focuses on integrating and showcasing the PhpStorm workflow and is current as of PhpStorm version 2017.1.4.
+
+## Create a WordPress site
+
+If you do not already have one, [create a new WordPress site](/docs/launch-wordpress/) on Pantheon.
```

Comment:
Is this site intended to be used exclusively for general plugin/theme development?